src/queries/pointsOfInterestAndTours.js

```diff
@@ -0,0 +1,189 @@
+import gql from 'graphql-tag';
+
+export const GET_POINTS_OF_INTEREST_AND_TOURS = gql`
+  query PointsOfInterestAndTours(
+    $limit: Int
+    $orderPoi: PointsOfInterestOrder
+    $orderTour: ToursOrder
+  ) {
+    pointsOfInterest(limit: $limit, order: $orderPoi) {
+      id
+      name
+      category {
+        name
+      }
+      description
+      mediaContents {
+        contentType
+        sourceUrl {
+          url
+        }
+      }
+      addresses {
+        city
+        street
+        zip
+      }
+      contact {
+        email
+        phone
+        lastName
+      }
+      webUrls {
+        url
+      }
+    }
+
+    tours(limit: $limit, order: $orderTour) {
+      id
+      name
+      category {
+        name
+      }
+      description
+      mediaContents {
+        sourceUrl {
+          url
+        }
+      }
+      dataProvider {
+        logo {
+          url
+        }
+        name
+      }
+    }
+  }
+`;
+
+export const GET_POINT_OF_INTEREST_AND_TOUR = gql`
```

Comment:
this makes no sense. reading it out: "let get us one point of interest and tour". there cannot be 1 poi and tour logically. there are at least 2 (1 poi and 1 tour), and therefore we would use the upper query.

need to remove this query.